### PR TITLE
Bug 442 fixing

### PR DESCRIPTION
**Pull request by andryspo**
added front-end  date validation, for user form,

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/user/my-bookings.jsp [hunk 1]
```diff
@@ -7,23 +7,29 @@
 <%@ taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
 
 <link rel='stylesheet' href='resources/css/user-my-report.css'>
+
 <script src="resources/js/myBookings.js"></script>
 <script src="resources/js/printMyBookings.js"></script>
 
+
+
+
```
Comment (vplytan-amplify) [inline src/main/webapp/WEB-INF/views/user/my-bookings.jsp]: ne need to have so many blank lines

File: src/main/webapp/WEB-INF/views/user/my-bookings.jsp [hunk 2]
```diff
@@ -65,3 +71,4 @@
 
     </div>
 </div>
+<script src="resources/js/validation-date.js"></script>
```
Comment (vplytan-amplify) [inline src/main/webapp/WEB-INF/views/user/my-bookings.jsp]: add new line in the end of the file

File: src/main/webapp/resources/js/myBookings.js [hunk 3]
```diff
@@ -100,4 +101,4 @@ var tableToExcel = ( function() {
                       $('#itemsPerPage').val(10);
                       paginate();
                       }
-          })()
\ No newline at end of file
+          })();
```
Comment (vplytan-amplify) [inline src/main/webapp/resources/js/myBookings.js]: new line

File: src/main/webapp/resources/js/validation-date.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        nowYear = new Date().getFullYear();
+        month = toDateValue.split("-")[1];
+        day = toDateValue.split("-")[2];
+        $('#to').val(nowYear + "-" + month + "-" + day);
+        return false;
+    }
+    if((new Date(fromDateValue)) > (new Date(toDateValue))) {
+        document.getElementById("dateError").
+            innerHTML = "<br>`from date` cannot be greater than `to date`";
+        return false;
```
Comment (vplytan-amplify) [inline src/main/webapp/resources/js/validation-date.js]: avoid multiple returns

**Comment by hrybs** [comment]
# **+**

**Pull request closed without merging**